modules/azure/src/main/java/org/testcontainers/containers/AzuriteContainer.java

```diff
@@ -0,0 +1,529 @@
+package org.testcontainers.containers;
```

Comment:
```suggestion
package org.testcontainers.azure;
```

Reply:
Thank you! I am happy to move this to a new package. 

Just a question whether that will be odd if we leave the CosmosDB container in the old package and add the new code to a new package with a name that would be appropriate for both Azurite and CosmosDB? https://github.com/testcontainers/testcontainers-java/tree/9cca5368343342e149bc452c473c52445637466c/modules/azure/src/main/java/org/testcontainers/containers

I mean CosmosDB is Azure as well, it would be a reasonable question from any future developer to ask why is it not in the Azure package then?

Reply:
Thanks, makes sense, just wanted to make sure it is intentional!

Reply:
Done, thank you!